app/locale/fr_FR/Oyst_OneClick.csv

```diff
@@ -151,3 +151,5 @@
 
 "Unable to refund Oyst order %s (%s - %s).","Impossible de rembourser de la commande %s (%s - %s)."
 "There is already one credit memo in the queue.","Il y a déjà un remboursement en cours."
+
+"Reinitialize delay buffer","Réinitialiser le tampon de délai"
```

Comment:
EN : Reset the order confirmation time for each product addition
FR : https://github.com/oystparis/oyst-1click-magento/issues/247#issue-297108950